LibreNMS/OS/HarmonyEnhancedMc.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace LibreNMS\OS;
+
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessRssiDiscovery;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessSnrDiscovery;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessPowerDiscovery;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessErrorsDiscovery;
+use LibreNMS\Device\WirelessSensor;
+use LibreNMS\OS;
+
+class HarmonyEnhancedMC extends OS implements WirelessRssiDiscovery, WirelessSnrDiscovery, WirelessPowerDiscovery, WirelessErrorsDiscovery
+{
+    public function discoverWirelessRssi()
+    {
+        $oids = snmpwalk_cache_oid($this->getDevice(), 'mwrEmcRadioRSL', array(), 'MWR-RADIO-MC-MIB', null, '-Ob');
+        $sensors = array();
+        foreach ($oids as $index => $entry) {
+            $sensors[] = new WirelessSensor(
+                'rssi',
+                $this->getDeviceId(),
+                '.1.3.6.1.4.1.7262.4.5.12.203.1.1.5.' . $index,
+                'harmony_enhanced_mc',
+                $index,
+                'RSL Radio ' .$index,
+                'null',
```

Comment:
this should just be null, not 'null', same with the next one and the divisor (10 instead of '10')